cores/esp8266/core_esp8266_main.cpp

```diff
@@ -156,11 +156,25 @@ void init_done() {
  * Peripherals (except for SPI0 and UART0) are not initialized.
  * This function does not return.
  */
+
```

Comment:
Please add a descriptive comment about what the #define means (explain it's either WPS or stack optimization, but not both), or at least reference the issue.

Reply:
Would this (above, below) be a clear enough explanation with its references ?